lib/resources/inetd_conf.rb

```diff
@@ -32,19 +36,8 @@ def method_missing(name)
 
     def read_params
       return @params if defined?(@params)
+      content = read_file_content(@conf_path)
```

Comment:
Same here, lets just use `@content` in this method as we already have the info. Or remove the `@content` from the init.